.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token:
```

Comment:
I think this file is not necessary. From the doc:
> This is only needed for private repos and should be kept secret – anyone could use it to submit coverage data on your repo’s behalf.

Reply:
No. this file is required to generate coverage badge number. We use `coveralls` package, this file is storage a token and we use this token to upload coverage to their system. You can see our coverage badge in here https://coveralls.io/github/tomochain/tomomaster?branch=master

Reply:
I feel like it's needed when run locally but not on a CI like Travis:
https://github.com/nickmerwin/node-coveralls#running-locally
I'm not sure that we need, can we just try to change it see if it still work without it?

Reply:
Ah nice, so we can remove it as it's not needed in the final workflow right? :+1:

Reply:
You can see it in my repo https://github.com/khaihkd/tomomaster on the top README file.

Reply:
Ok, anyway, we can change it later if it's not needed